packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Configuration.cs

```diff
@@ -144,6 +145,25 @@ private static class Options
         private string? _projectDirectory;
         internal string ProjectDirectory => _projectDirectory ??= Path.Combine(OutputDirectory, "src");
 
+        private string? _generatedOutputDirectory;
+        internal string GeneratedSourceOutputDirectory => _generatedOutputDirectory ??= ParseGeneratedSourceOutputPath();
+
+        private string? _generatedTestOutputDirectory;
+        internal string GeneratedTestOutputDirectory => _generatedTestOutputDirectory ??= Path.Combine(OutputDirectory, "tests", GeneratedFolderName);
+
+        private string ParseGeneratedSourceOutputPath()
```

Comment:
what do you mean we should not need this?

Reply:
the method returns the same value as ProjectDirectory property?  This is a copy paste from autorest that isn't needed anymore because we don't allow for multiple commandline input formats.

The input on the commandline is the OutputDir and the ProjectDir is always `OutputDir/src`

Reply:
OK. Let me change this to just concat `Generated` to `ProjectDirectory`

Reply:
I also added another property `TestProjectDirectory` which corresponds to `OutputDirectory/tests`
And now everything should be uniformed:
- ProjectDirectory=OutputDirectory/src
- TestProjectDirectory = OutputDirectory/tests
- ProjectGeneratedDirectory = ProjectDirectory/Generated
- TestGeneratedDirectory = TestProjectDirectory/Generated

What do you think?